src/client/components/SearchAPI/Fetcher.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+
+const baseUrl = 'https://api.github.com/search/users?q=';
+
+export const Fetcher = ({ search, render }) => {
```

Comment:
instead of adding `render` as a prop you could just return the data so you don't have to pass something in here.

Will show in another comment how that will work when returning

```suggestion
export const Fetcher = ({ search }) => {
```